newdle/schemas.py

```diff
@@ -126,6 +156,13 @@ def validate_timeslots(self, v):
             raise ValidationError('Time slots are not unique')
 
 
+class UpdateNewdleSchema(NewNewdleSchema):
+    participants = fields.List(
+        fields.Nested(NewParticipantSchema, unknown=EXCLUDE), missing=[]
```

Comment:
`missing=[]` seems very dangerous here... it works fine because just like required it's ignored with `partial=True` but if someone removed that, it would delete all participants when finalizing an newdle. I think just removing the `missing` kwarg here doesn't change the behavior but avoids this possible pitfall.

Reply:
That's completely right, this went through so many iterations that I eventually forgot to remove it when copying, good point.